kanidmd/src/lib/config.rs

```diff
@@ -115,7 +115,7 @@ impl Configuration {
                 self.tls_config = Some(TlsConfiguration { chain, key })
             }
             _ => {
-                error!("Invalid TLS configuration - must provide chain and key!");
+                eprintln!("Invalid TLS configuration - must provide chain and key!");
```

Comment:
May be add prefix `ERROR: ` so it is more inline with the other warning/error messages.

For the rest it looks good.

Reply:
Good point. I'll change that and merge. Thanks for reviewing and for reporting the issue!